src/main/java/com/fulcrumgenomics/util/PickIlluminaIndicesCommand.java

```diff
@@ -318,6 +319,7 @@ protected int execute() {
             if (first.minEditDistance() > lastMinEdit) {
                 lastMinEdit = first.minEditDistance();
                 log.info("There are " + count + " indices with a minimum edit distance of " + lastMinEdit);
+                if (count == N_INDICES) break;
```

Comment:
This is a corner case bug.  If we have N candidate indices and also want N candidate indices, then without this we remove one and so return `N-1` indices.  This only mattered when the # of candidate indices was the same as the target # of indices, which is non-sensical without providing a list of indices from which to pick